src/naive/internals.rs

```diff
@@ -467,11 +466,28 @@ impl fmt::Debug for Mdf {
     }
 }
 
+/// Create a `Weekday` from an `u32`, with Monday = 0.
+/// Infallible, takes `n % 7`.
```

Comment:
The `takes n % 7` comment is no longer correct, right?

Reply:
The intention was to say it is infallible, because it takes any `n` and does modulus 7. I'll see if I can word it better.